packages/commcare/test/index.js

```diff
@@ -319,6 +319,54 @@ describe('getCases', () => {
       },
     ]);
   });
+
+  it('should fetch cases with pagination', async () => {
+    const objects = [
+      { case_id: '123' },
+      { case_id: '456' },
+      { case_id: '789' },
+    ];
+    testServer
+      .intercept({
+        path: /\/a\/my-domain\/api\/v0\.5\/case/,
+        method: 'GET',
+      })
+      .reply(200, req => {
+        const offset = req.query.offset;
+        const next =
+          req.query.offset < objects.length - 1
+            ? `offset=${offset + 1}&limit=1`
+            : null;
+        return {
+          meta: {
+            limit: 1,
+            next: next,
+            offset: offset,
+            previous: null,
+            total_count: 3,
+          },
+          objects: [objects[offset]],
+        };
+      })
+      .times(4);
+
+    const state = {
+      configuration: {
+        hostUrl,
+        domain,
+        appId: app,
+        username: 'user',
+        password: 'password',
+      },
+    };
+
+    const { data, response } = await execute(
+      get('case', { limit: 1, offset: 0 })
```

Comment:
If we're testing auto-pagination, we should not pass an offset here.

Reply:
This comment still stands, and if this test is passing we have a problem. The `get()` function here should not paginate.